### PR TITLE
Adding Maps and Mapmerger

### PR DESCRIPTION
**Pull request by altesismein**
I added the maps, and configured the mapmerger.

Read the instructions in tools/mapmerge

### CODE REVIEW

File: tools/mapmerge/Prepare Maps.bat [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @echo off
-cd ../../_maps/
+cd ../../_maps/map_files/Majid
```
Comment (tigercat2000) [inline tools/mapmerge/Prepare Maps.bat]: this isn't necessary, the mapmerger will work correctly for every sub-folder

File: tools/mapmerge/Run Map Merge.bat [hunk 2]
```diff
@@ -1,4 +1,4 @@
 @echo off
-call java -jar MapMerge.jar "../../_maps/" /wait
+call java -jar MapMerge.jar "../../_maps/map_files/Majid" /wait
```
Comment (tigercat2000) [inline tools/mapmerge/Run Map Merge.bat]: same here

**Comment by altesismein** [comment]
Ok, i reverted my changes to the Mapmerger

**Comment by tigercat2000** [comment]
This can be merged as soon as travis is done, no real reason to wait.

**Pull request merged**